core/src/main/java/com/nisovin/magicspells/spells/PlayerMenuSpell.java

```diff
@@ -153,12 +161,34 @@ public boolean castFromConsole(CommandSender sender, String[] args) {
 		return true;
 	}
 
+	private ItemStack createItem(String path, String defaultName) {
+		ItemStack item = null;
+		MagicItem magicItem = null;
+
+		if (MagicItems.getMagicItemFromString(getConfigString(path, "")) != null) {
+			magicItem = MagicItems.getMagicItemFromString(getConfigString(path, "")).clone();
+		}
+
+		if (magicItem == null) {
+			item = new ItemStack(Material.GREEN_WOOL);
+
+			ItemMeta itemMeta = item.getItemMeta();
+			itemMeta.displayName(Util.getMiniMessage("&6" + defaultName));
+
+			item.setItemMeta(itemMeta);
+		} else {
+			item = magicItem.getItemStack();
+		}
+
+		return item;
+	}
```

Comment:
You should be cloning the item from the magic item, not the magic item itself. Also, unnecessary to use `Util#getMiniMessage` - build the component directly.

What I generally meant was something more like:
```java
private ItemStack createItem(String path, String defaultName) {
	MagicItem magicItem = MagicItems.getMagicItemFromString(getConfigString(path, null));
	if (magicItem != null) return magicItem.getItemStack().clone();

	ItemStack item = new ItemStack(Material.GREEN_WOOL);
	ItemMeta meta = item.getItemMeta();

	meta.displayName(Component.text(defaultName).color(NamedTextColor.GOLD));
	item.setItemMeta(meta);

	return item;
}
```